ntc_templates/templates/huawei_smartax_display_ont_info_summary_sn.textfsm

```diff
@@ -1,17 +1,26 @@
+Value Key,Filldown PORT (\S+)
 Value Key ONT_ID (\d+)
 Value SERIAL_NUMBER (\w+)
 Value ONT_MODEL_TYPE (\S+)
 Value DISTANCE_M (\d+|-)
-Value RX_TX_POWER_DBM (-?\d+\.\d+\/-?\d+\.\d+|\S+)
-Value DESCRIPTION (\S+)
+Value RX_POWER_DBM (-?\d+\.\d+)
+Value TX_POWER_DBM (-?\d+\.\d+)
+Value DESCRIPTION (\S.+\S)
 
 Start
+  ^\s*Command\s*is\s*being\s*executed\.\s*Please\s*wait\s*$$
+  ^\s*In\s*port\s*${PORT},\s*the\s*total\s*of\s*ONTs\s*are:\s*\d+,\s*online:\s*\d+\s*$$
+  ^\s*ONT\s*Run\s*Last\s*Last\s*Last\s*$$
+  ^\s*ID\s*State\s*UpTime\s*DownTime\s*DownCause\s*$$
+  ^\s*\d+\s*\S+\s*(-|\S+\s\S+)\s*(-|\S+\s*\S+)\s*\S+\s*$$
   ^\s+-
```

Comment:
Similar to the other template above...
This will match the horizontal separator lines and is unlikely to match much else.

```suggestion
  ^\s+-+\s*$$
```